.changeset/orange-hairs-fold.md

```diff
@@ -0,0 +1,9 @@
+---
+"@siteimprove/alfa-rules": minor
+"@siteimprove/alfa-json": minor
+"@siteimprove/alfa-tree": minor
+"@siteimprove/alfa-act": minor
+"@siteimprove/alfa-dom": minor
+---
+
+**Added:** Verbosity option has been added to JSON serialization of alfa-tree `Node` and to `Outcome` and `Group`.
```

Comment:
We need the symmetrical changeset that `Node` builders now accept an optional `serializationID`.